Assets/__Scripts/FileOperationAPIWrapper.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
```

Comment:
Is this an open source file? If so, please include license info/credits.

Reply:
This is a fair question, as far as I can tell the original version of this was posted here https://stackoverflow.com/questions/3282418/send-a-file-to-the-recycle-bin

Which would imply copyright, however I'm not sure there's anything in this file that doesn't derive from the Microsoft dll (and therefore microsoft's documentation https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-ifileoperation-setoperationflags , https://docs.microsoft.com/en-gb/windows/win32/api/shellapi/ns-shellapi-shfileopstructa ) which means that it's not itself a copyright-able work as it *has* to be written this way.

Reply:
I can strip out most of the file I think if you'd prefer

Reply:
I think its fine for the reasons you've already said. If it was me personally, I'd comment the URL to the stackoverflow post but that would be up to you.